src/services/railgun/quick-sync/V2/__tests__/quick-sync-events-graph-v2.test.ts

```diff
@@ -145,8 +145,35 @@ describe('quick-sync-events-graph-v2', () => {
 
     const shouldThrow = true;
     assertContiguousCommitmentEvents(eventLog.commitmentEvents, shouldThrow);
-  }).timeout(90_000);
+  }).timeout(200_000);
+  /*
+  it('[V2] Should make sure Graph V2 query has no data gaps in commitments - Polygon Mumbai', async function run() {
+    if (!isV2Test()) {
+      this.skip();
+      return;
+    }
+
+    const eventLog = await quickSyncEventsGraph(
+      txidVersion,
+      POLYGON_MUMBAI_CHAIN,
+      0,
+    );
+    expect(eventLog).to.be.an('object');
+    expect(eventLog.commitmentEvents).to.be.an('array');
+    expect(eventLog.commitmentEvents.length).to.be.at.least(
+      EXPECTED_COMMITMENT_GROUP_EVENTS_POLYGON_MUMBAI,
+    );
+    expect(eventLog.nullifierEvents.length).to.be.at.least(
+      EXPECTED_NULLIFIER_EVENTS_POLYGON_MUMBAI,
+    );
+    expect(eventLog.unshieldEvents.length).to.be.at.least(
+      EXPECTED_UNSHIELD_EVENTS_POLYGON_MUMBAI,
+    );
 
+    const shouldThrow = true;
+    assertContiguousCommitmentEvents(eventLog.commitmentEvents, shouldThrow);
+  }).timeout(45000);
+  */
```

Comment:
Can you just delete this commented code? It's not needed anymore since Polygon Mumbai is a dead testnet.